siphon/simplewebservice/wyoming.py

```diff
@@ -93,14 +97,15 @@ def _get_data(self, time, site_id):
         latitude = float(lines[4].split(':')[1].strip())
         longitude = float(lines[5].split(':')[1].strip())
         elevation = float(lines[6].split(':')[1].strip())
+        pw          = float(lines[-1].split(':')[1].strip())
```

Comment:
Don’t worry about trying to line things up, just put a single space on either side of the equal sign.